src/main/java/MironovAnastasia/lesson1.java

```diff
@@ -0,0 +1,112 @@
+package IvanTrembaci.src.main.java.MironovAnastasia;
+
+import java.sql.SQLOutput;
+import java.util.Scanner;
+
+public class lesson1 {
```

Comment:
The same class lesson1 I see in 3 PRs
Please remove it from Pull requests where it is not needed, otherwise - conflict.